src/envoy/utils/mixer_control.cc

```diff
@@ -100,5 +102,67 @@ Grpc::AsyncClientFactoryPtr GrpcClientFactoryForCluster(
   return std::make_unique<EnvoyGrpcAsyncClientFactory>(cm, service);
 }
 
+/** example node
+   "node": {
+     "id":
+"sidecar~10.36.0.15~fortioclient-84469dc8d7-jbbxt.service-graph~service-graph.svc.cluster.local",
+     "cluster": "fortioclient",
+     "metadata": {
+      "ISTIO_VERSION": "1.0.1",
+      "POD_NAME": "fortioclient-84469dc8d7-jbbxt",
+      "istio": "sidecar",
+      "INTERCEPTION_MODE": "REDIRECT",
+      "ISTIO_PROXY_VERSION": "1.0.0",
+      "ISTIO_PROXY_SHA": "istio-proxy:2656f34080413d3aec444aa659cc78057508c57b"
+     },
+     "build_version": "0/1.8.0-dev//RELEASE"
+    },
+
+    ==> uid: kubernetes://fortioclient-84469dc8d7-jbbxt.service-graph
+    reporter == uid
+    namespace
+    IP_Address only for inbound.
+**/
+const LocalAttributes GenerateLocalAttributes(
```

Comment:
The expected format should be clearly documented in the comment.  This is very error pron. Not sure if how often Pilot agent changes this format,   May have to add comment in the Pilot agent